package.json

```diff
@@ -1,9 +1,8 @@
 {
   "name": "@sanity/hierarchical-document-list",
-  "version": "1.1.0",
+  "version": "1.2.5",
```

Comment:
can you revert the version update? we'll update the version number as part of the release process